.github/CODEOWNERS

```diff
@@ -1 +1 @@
-*   @xluo-aws @gaobinlong @Hailong-am @SuZhou-Joe @zhichao-aws @yuye-aws @getsaurabh02 @sbcd90 @praveensameneni @amsiglan @AWSHurneyt
\ No newline at end of file
+*   @xluo-aws @gaobinlong @Hailong-am @SuZhou-Joe @zhichao-aws @yuye-aws @getsaurabh02 @sbcd90 @praveensameneni @amsiglan @AWSHurney @riysaxen-amzn
```

Comment:
@riysaxen-amzn 
Looks like the `t` was removed from the end of my username. Could you add it back?

Reply:
addressed